pyproject.toml

```diff
@@ -10,6 +10,7 @@ authors = [
 	{ name="Johannes Gäßler", email="johannes.gaessler@cern.ch" },
 	{ name="Cedric Verstege", email="cedric.verstege@cern.ch" },
 	{ name="Günter Quast", email="G.Quast@kit.edu" },
+	{ name="Michael Hohenstein", email="michael@hohenste.in" },
```

Comment:
We have to define what our policy here will be. You are definitely shown as a contributor on the GitHub page, since you have commits in the code.
The authors of the python package might be more prone to receiving theoretical question about fitting in general. Not sure if we want to include every contributor here or keep this limited. This needs to be discussed.